src/pages/Landing.tsx

```diff
@@ -1,10 +1,11 @@
-import { FooterSection } from '@components';
+import { FooterSection, About } from '@components';
 import { Navbar } from '@components';
 
 const Landing: React.FC = () => {
     return (
-        <div>
+        <div className="grid grid-cols-1">
```

Comment:
```suggestion
        <div>
```
This does not do anything to the layout though :<

Also, the svg component on the far right causes overflow, may fix it with overflow hidden to the about section itself

![Screenshot 2024-02-25 at 9 09 22 PM](https://github.com/LaurierHawkHacks/Landing/assets/98545971/e09ff407-9983-4799-b6d3-38020ad058c1)

Reply:
You're absolutely right! It did something earlier when I was messing around with the components. I forgot to take it out 😅 good catch!!